version: '3.8'services:  cfour_backend_dev:    build: .    container_name: cfour_backend_develop    ports:      - "8000:8000"    environment:      - DEBUG=True      - PYTHONDONTWRITEBYTECODE=1    command: [ "python", "manage.py", "runserver", "0.0.0.0:8000" ]    volumes:      - .:/code    networks:      - cfour_network#  cfour_minio_dev:#    image: minio/minio:latest#    container_name: lexico-develop-minio-1#    ports:#      - "9000:9000"#      - "9001:9001"#    volumes:#      - ./minio-data:/data#    networks:#      - cfour_network##    environment:#      - MINIO_ROOT_USER=admin#      - MINIO_ROOT_PASSWORD=admin#      - MINIO_ADDRESS=:9000#      - MINIO_CONSOLE_ADDRESS=:9001#    command: server /data --console-address :9001#    healthcheck:#      test: [ "CMD", "curl", "-f", "http://127.0.0.1:9000/minio/health/live" ]#      interval: 30s#      timeout: 20s#      retries: 3networks:  cfour_network:    external: true